tcms/testruns/views.py

```diff
@@ -1402,11 +1398,11 @@ def remove(self):
                 return HttpResponse(json.dumps(chk_perm))
 
             try:
-                for tr in self.trs:
-                    tr.remove_env_value(env_value=self.get_env_value(
+                for test_run in self.test_runs:
+                    test_run.remove_env_value(env_value=self.get_env_value(
                         request.GET.get('env_value_id')
                     ))
-            except:
+            except Exception:
```

Comment:
This is a bit strange. I think the original idea is to not crash in case removing env value fails. However as currently written this will exit the loop. OTOH you can do:
```
for:
    try:
    except Exception:
        pass # or continue
```

That has the effect of still ignoring the errors but at least allows us to cycle through the entire query set,